sdk/go/x/auto/local_workspace.go

```diff
@@ -136,7 +137,7 @@ func (l *LocalWorkspace) GetConfig(ctx context.Context, fqsn string, key string)
 	if err != nil {
 		return val, errors.Wrapf(err, "could not get config, unable to select stack %s", fqsn)
 	}
-	stdout, stderr, errCode, err := l.runPulumiCmdSync(ctx, "config", "get", key, "--show-secrets", "--json")
+	stdout, stderr, errCode, err := l.runPulumiCmdSync(ctx, "config", "get", key, "--json")
```

Comment:
`--show-secrets` was an intentional decision here (and below). The return type of `GetConfig` is a `ConfigValue` which has:

- `Value`: the plaintext context of the config
- `Secret`: a boolean flag indicating secretness

This allows users (who are presumably privileged) to handle secrets with the maximum flexibility possible without overcomplicating the API surface.

Reply:
There is no such flag on config get - it *always* returns the unencrypted value by default

Reply:
LOL good catch :)